gems/nokogiri/GHSA-v6gp-9mmm-c6p5.yml

```diff
@@ -0,0 +1,30 @@
+---
+gem: nokogiri
+ghsa: v6gp-9mmm-c6p5
+url: https://github.com/sparklemotion/nokogiri/security/advisories/GHSA-v6gp-9mmm-c6p5
+title: Out-of-bounds Write in zlib affects Nokogiri
+date: 2022-04-11
+description: "## Summary\n\nNokogiri v1.13.4 updates the vendored zlib from 1.2.11
```

Comment:
Please use `description: |` so that the text isn't an escaped double-quoted String, which makes it easier to read and edit.